### PR TITLE
ACP2E-346: [Documentation] Add final minimum(maximum) price calculation formula into number of documents

### PR DESCRIPTION
**Pull request by vkolesny**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

According to the Product Owner clarification:
https://jira.corp.magento.com/browse/MC-41581?focusedCommentId=1660777&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1660777

And according to the existing internal documentation:
https://wiki.corp.magento.com/display/SUSENG/Price+calculation+for+indexer#Pricecalculationforindexer-SimpleProductwithcustomoptions+tier(grouped)price

We should inform Merchants in the external documentation about minimum(maximum) product price calculation formula:
Final Min(Max) Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

Magento 2.4.x

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

Both Magento Open Source and Adobe Commerce

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

B2B extension, but only in scope of this document update:
https://docs.magento.com/user-guide/catalog/catalog-shared-pricing-structure.html

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/catalog/catalog-shared-pricing-structure.html
- https://docs.magento.com/user-guide/marketing/price-rules-catalog-create.html
- https://docs.magento.com/user-guide/catalog/product-price-group.html
- https://docs.magento.com/user-guide/catalog/product-price-special.html
- https://docs.magento.com/user-guide/catalog/product-price-tier.html

### CODE REVIEW

File: src/catalog/catalog-shared-pricing-structure.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Only categories with selected products appear in the [top navigation]({% link ca
 
 You can set custom pricing for each product individually or use the Action control to set custom pricing as a fixed amount or percentage for multiple product records.
 
-| Fixed | Specifies the final product price. For example, if you enter a fixed price of $10.00, the price in the storefront for the corresponding company is $10.00. |
+| Fixed | Specifies the final product price. For example, if you enter a fixed price of $10.00, the price in the storefront for the corresponding company is $10.00. **Note:** Minimum value among Base Price and entered Fixed value vil be selected as final product price |
```
Comment (leslietilling) [inline src/catalog/catalog-shared-pricing-structure.md:65]:
```suggestion
| Fixed | Specifies the final product price. For example, if you enter a fixed price of $10.00, the price in the storefront for the corresponding company is $10.00. **_Note:_** The minimum value between the Base Price and the entered Fixed value is used as the final product price. |
```

File: src/catalog/product-price-group.md [hunk 2]
```diff
@@ -54,3 +54,6 @@ _Customer Group Discount_
 
     ![]({% link images/images/storefront-cart-price-group-discount.png %}){: .zoom}
     _Group Price in Shopping Cart_
+
+{:.bs-callout-tip}
+The **_final_** product price is calculated as the **_minimum_** relevant price, by the formula: <br/>Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)
```
Comment (leslietilling) [inline src/catalog/product-price-group.md:59]:
```suggestion
The **_final_** product price is calculated as the **_minimum_** relevant price, using the following formula: <br/>`Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)`
```

File: src/catalog/product-price-special.md [hunk 3]
```diff
@@ -154,3 +154,6 @@ If the special price does not appear correctly in the storefront on both the cat
 1. On the _Admin_ sidebar, go to **System** > **Cache Management**.
 
 1. Click <span class="btn">Flush Magento Cache</span>.
+
+{:.bs-callout-tip}
+The **_final_** product price is calculated as the **_minimum_** relevant price, by the formula: <br/>Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)
```
Comment (leslietilling) [inline src/catalog/product-price-special.md:159]:
```suggestion
The **_final_** product price is calculated as the **_minimum_** relevant price, using the following formula: <br/>`Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)`
```

File: src/catalog/product-price-tier.md [hunk 4]
```diff
@@ -57,3 +57,6 @@ The prices in the storefront take precedence from the highest to the lowest quan
 1. To add another set of tier pricing for a different website or customer group, repeat the previous steps.
 
 1. When complete, click <span class="btn">Done</span> and then <span class="btn">Save</span>.
+
+{:.bs-callout-tip}
+The **_final_** product price is calculated as the **_minimum_** relevant price, by the formula: <br/>Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)
```
Comment (leslietilling) [inline src/catalog/product-price-tier.md:62]:
```suggestion
The **_final_** product price is calculated as the **_minimum_** relevant price, using the following formula: <br/>`Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)`
```

File: src/marketing/price-rules-catalog-create.md [hunk 5]
```diff
@@ -112,7 +112,7 @@ Most of the available conditions are based on existing attribute values. To appl
    |Adjust final price to discount value|Sets the final price to a fixed, discounted amount. For example: Enter 20 in Discount Amount for a final price of $20.00.|
 
     {:.bs-callout-info}
-    _Regular price_ refers to the base product price without any advanced pricing (special/tier/group) or promotional discounts. _Final price_ refers to the discounted price that appears in the shopping cart.
+    _Regular price_ refers to the base product price without any advanced pricing (special/tier/group) or promotional discounts. _Final price_ refers to the discounted price that appears in the shopping cart. <br/>The **_final_** product price is calculated as the **_minimum_** relevant price, by the formula: <br/>Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)
```
Comment (leslietilling) [inline src/marketing/price-rules-catalog-create.md:115]:
```suggestion
    _Regular price_ refers to the base product price without any advanced pricing (special/tier/group) or promotional discounts. _Final price_ refers to the discounted price that appears in the shopping cart. <br/>The **_final_** product price is calculated as the **_minimum_** relevant price, using the following formula: <br/>`Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)`
```

File: src/marketing/price-rules-catalog-create.md [hunk 6]
```diff
@@ -204,7 +204,7 @@ Specifies the conditions that must be met before the catalog price rule goes int
...
+|Apply|Determines the type of calculation that is applied to the purchase. Options: <br/>**Apply as percentage of original** - Discounts item by subtracting a percentage of the regular price. <br/>**Apply as fixed amount** - Discounts item by subtracting a fixed amount from the regular price. <br/>**Adjust final price to this percentage** - Adjusts the final price by a percentage of the regular price. <br/>**Adjust final price to discount value** - Sets the final price to a fixed, discounted amount. <br/><br/>**_Note:_** Regular price refers to the base product price without any advanced pricing (special/tier/group) or promotional discounts. Final price refers to the discounted price that appears in the shopping cart. <br/>The **_final_** product price is calculated as the **_minimum_** relevant price, by the formula: <br/>Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)|
```
Comment (leslietilling) [inline src/marketing/price-rules-catalog-create.md:207]:
```suggestion
|Apply|Determines the type of calculation that is applied to the purchase. Options: <br/>**Apply as percentage of original** - Discounts item by subtracting a percentage of the regular price. <br/>**Apply as fixed amount** - Discounts item by subtracting a fixed amount from the regular price. <br/>**Adjust final price to this percentage** - Adjusts the final price by a percentage of the regular price. <br/>**Adjust final price to discount value** - Sets the final price to a fixed, discounted amount. <br/><br/>**_Note:_** Regular price refers to the base product price without any advanced pricing (special/tier/group) or promotional discounts. Final price refers to the discounted price that appears in the shopping cart. <br/>The **_final_** product price is calculated as the **_minimum_** relevant price, using the following formula: <br/>`Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)`|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just slight editorial tweaks and formatting for formula

File: src/catalog/catalog-shared-pricing-structure.md (hunk 1 above)
Comment (vkolesny) [reply src/catalog/catalog-shared-pricing-structure.md]: Committed

File: src/catalog/product-price-special.md (hunk 3 above)
Comment (vkolesny) [reply src/catalog/product-price-special.md]: Committed

File: src/marketing/price-rules-catalog-create.md (hunk 5 above)
Comment (vkolesny) [reply src/marketing/price-rules-catalog-create.md]: Committed

File: src/catalog/product-price-group.md (hunk 2 above)
Comment (vkolesny) [reply src/catalog/product-price-group.md]: Committed

File: src/catalog/product-price-tier.md (hunk 4 above)
Comment (vkolesny) [reply src/catalog/product-price-tier.md]: Committed

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**